packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -932,6 +949,7 @@ contract VotingV2 is
         if (slash + int256(voterStake.activeStake) > 0)
             voterStake.activeStake = uint256(int256(voterStake.activeStake) + slash);
         else voterStake.activeStake = 0;
+        if (voterStake.unappliedSlash != 0) voterStake.unappliedSlash = 0;
```

Comment:
We can test this, but I think it might be cheaper to just do (since we'll only get charged for a read either way I think!):

```
voterStake.unappliedSlash = 0;
```